Libraries/Inspector/resolveRelativeSizes.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const Dimensions = require('Dimensions');
+
+/**
+ * Resolves relative sizes (percentages and auto) in a style object.
+ */
+function resolveRelativeSizes(style: Object) {
```

Comment:
Could you change this flow type to have the correct object type information? Something like: 

```js
type Style = {
  height: number,
  top: number,
  …
}
```

?